README.md

```diff
@@ -2,7 +2,11 @@
 
 Pecorino is a rate limiter based on the concept of leaky buckets, or more specifically - based on the [generic cell rate](https://brandur.org/rate-limiting) algorithm. It uses your DB as the storage backend for the throttles. It is compact, easy to install, and does not require additional infrastructure. The approach used by Pecorino has been previously used by [prorate](https://github.com/WeTransfer/prorate) with Redis, and that approach has proven itself.
 
-Pecorino is designed to integrate seamlessly into any Rails application using a PostgreSQL or SQLite database (at the moment there is no MySQL support, we would be delighted if you could add it).
+Pecorino is designed to integrate seamlessly into any Rails application, and will use either:
+
+* A memory store (good enough if you have just 1 process)
+* A PostgreSQL or SQLite database (at the moment there is no MySQL support, we would be delighted if you could add it)
+* A Redis instance
 
```

Comment:
**nitpick:** might be a good idea to explain how to redefine adapter - in case of redis, as example, adapter will not be inherited from ActiveRecord and requires a manual configuration.

Reply:
None of them inherit from ActiveRecord anymore, some just use the ActiveRecord classes as means of passing connection configuration (and to piggyback on escaping)

Reply:
> None of them inherit from ActiveRecord anymore

Why are you saying that? there is a `default_adapter_from_main_database` method still defined.

https://github.com/cheddar-me/pecorino/pull/18/files#diff-3ae95d36dd505f2d90c28bab102aceb7c117caee072c2a5910685a35fb1f777bR49

But it would be great to add a section about how to configure adapter for pecorino and which settings are being configured by default.

Reply:
There is that method indeed (to let users have their adapter picked automatically), but nothing inherits anything there?..

Reply:
Sorry for the confusion, indeed wrong use of the word "inherits".